completion/_blinky

```diff
@@ -0,0 +1,146 @@
+#compdef blinky
+# Copyright (C) 2018 Paul Seyfert <pseyfert.mathphys@gmail.com>
+#
+# This software is distributed under the terms of the GNU General Public
+# Licence version 3 (GPL Version 3), copied verbatim in the file "LICENSE".
+
+# heavily inspired by the _pacman completion
+
+local -a _blinky_opts_all _blinky_opts_install _blinky_opts_build
+
```

Comment:
no visible: I anticipate the dropping of --local-path

Reply:
this was done in 7b8d8e92cfd8ea58a44599470f5f19b85ec98206 so no longer *anticipating*